keras_cv/datasets/pascal_voc/segmentation.py

```diff
@@ -406,10 +409,13 @@ def _build_dataset_from_metadata(metadata):
 def _build_sbd_dataset_from_metadata(metadata):
     img_filepath = metadata["image/file_path"]
     cls_filepath = metadata["segmentation/class/file_path"]
-    obj_file_path = metadata["segmentation/object/file_path"]
+    obj_filepath = metadata["segmentation/object/file_path"]
 
     def md_gen():
-        for img_fp, cls_fp, obj_fp in zip(img_filepath, cls_filepath, obj_file_path):
+        c = list(zip(img_filepath, cls_filepath, obj_filepath))
+        random.shuffle(c)
```

Comment:
Is it possible to have the shuffle after the dataset is built? or you would like the original dataset to be in random order?

Reply:
I tested only shuffling before the generator vs shuffling at each time. Apparently the latter gives ~10+ MeanIOU